Web/Element/HeroImage.hs

```diff
@@ -0,0 +1,55 @@
+module Web.Element.HeroImage where
+
+import Web.View.Prelude
+
+import Web.Element.ElementWrap
+import Web.Element.Types
+import Web.Element.Button
+
+render :: RenderHeroImage -> Html
+render record@RenderHeroImage {title, maybeSubtitle, maybeButton, imageUrl} =
+     titleWrapped
+        ++ subTitleWrapped
+        ++ renderedButton
```

Comment:
We can convert this to a list of [Maybe Html], and then filter out everything that's a `Nothing`

```
[ titleWrapped
, maybeSubTitleWrapped
, maybe...
]
|> catMaybes
|> mconcat
```